platform/pensando/dsc-drivers/debian/control

```diff
@@ -0,0 +1,14 @@
+Source: ionic
+Section: main
+Priority: extra
+Maintainer: Guohan Lu <gulv@microsoft.com>
```

Comment:
can we get a proper name for maintainer? someone from amd? not sure I am the maintainer for this package.

Reply:
Done